openedx/features/edly/models.py

```diff
@@ -35,3 +37,25 @@ class Meta:
 
     def __str__(self):
         return '{name}: ({slug})'.format(name=self.name, slug=self.slug)
+
+
+class EdlyUserProfile(models.Model):
+    """
+    User profile model for Edly users.
+    """
+    user = models.OneToOneField(User, unique=True, db_index=True, related_name='edly_profile', on_delete=models.CASCADE)
+    edly_sub_organizations = models.ManyToManyField(EdlySubOrganization)
+
+    @property
+    def get_linked_edly_sub_organizations(self):
+        """
+        Helper method to get list of slugs of edly sub organizations of a user.
+
+        Returns:
+            List of edly sub organizations slugs
+        """
+        edly_sub_org_slugs = []
+        if self.edly_sub_organizations:
+            edly_sub_org_slugs = [edly_sub_org.slug for edly_sub_org in self.edly_sub_organizations.all()]
```

Comment:
use `values_list` to get a list of all slugs also this method should be two liner
```python
edly_sub_org_slugs = self.edly_sub_organizations.values_list('slug', flat=True)
return edly_sub_org_slugs
```

Reply:
Done